packages/back-end/src/app.ts

```diff
@@ -471,7 +471,7 @@ app.delete("/datasource/:id", datasourcesController.deleteDataSource);
 
 // Events
 app.use("/events", eventsRouter);
-app.use("/event-webhooks", eventWebHooksRouter);
+app.use(eventWebHooksRouter);
```

Comment:
The whole URL is defined in the router. I thought this was easier to read.

Reply:
I can see pros and cons of both approaches.  It is nice to be able to see the entire API route in a single string - makes it easy to Ctrl+F.  On the other hand, it's nice to see all the top-level routes at a glance in a single file.

I'm fine keeping it how it is in this PR.  It's a really easy refactor to change it later if we feel like that's a better approach.